grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.html

```diff
@@ -97,6 +97,14 @@ <h2>NOTES</h2>
 <p>
 The query string sent to ESA can be shown with <b>--verbose</b>.
 
+<h3>Long Term Archive (LTA)</h3>
+Access to the product URL for data that are no longer available online will automatically trigger
+the retrieval of the data from the LTA, the actual download may be initiated once the data are restored
+(within minutes to hours). Full details are available in the updated
+<a href="https://scihub.copernicus.eu/userguide/" target="_blank">user guides</a>. 
+
```

Comment:
In this <a href="https://github.com/sentinelsat/sentinelsat/issues/358#issuecomment-615895388" target="_blank">issue of sentinelsat</a> , it is written:<br>
<em>Please be advised that a user quota on the maximum number of requests per hour per user is set. Note that if you
<ul>
  <li>retrieve data directly from the GUI, you can retrieve only 1 product per half hour while</li>
  <li>from APIHub 20 products each 12h</li>
</em>